sdk.yaml

```diff
@@ -281,6 +281,10 @@ Realtime:
           (optional functionality).
   Connection:
     .specification: RTC2
+    Get Identifier:
+      .specification: RTN8
+      .synopsis: |
+        Get the `id` of the active connection, which is a public string provided by the Ably service and is unique for this connection.
```

Comment:
Does the connection need to be active to have an ID? For example, if the client connects, then gets disconnected, I think it still has an ID which continues to be valid once the client reconnects.

Reply:
I've adjusted the wording: https://github.com/ably/features/pull/23/commits/c80be8b40704f8d0465dcc150b8d6fe48b6e25a6

I'm going to resolve this conversation on @lmars' behalf, in order that this pull request can merge.